package.json

```
@@ -56,8 +56,14 @@
     "opn": "^5.4.0"
   },
   "devDependencies": {
+    "husky": "^1.2.1",
     "jest": "^23.6.0",
     "markdown-magic": "^0.1.25",
     "standard": "^12.0.1"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run test && npm run markdown"
```

running `npm run markdown` is not needed, as we already validate in `npm run test`.
```suggestion
      "pre-commit": "npm run test"
```

---

Is there a way for telling husky to not include `unstage` files while running hooks?

---

If y'all can come to a conclusion on this, this can be merged 👍